### PR TITLE
Added exit game from button + fix rg3d dependency

### PR DESCRIPTION
**Pull request by git2013vb**

### CODE REVIEW

File: src/rg3d/beginning/getting_started.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ To begin using the engine, just add following lines to your `Cargo.toml`:
 
 ```toml
 [dependencies]
-rg3d = { git = "https://github.com/rg3dengine/rg3d" }
+rg3d = { git = "https://github.com/rg3dengine/rg3d" }
```
Comment (mrDIMAS) [inline src/rg3d/beginning/getting_started.md:10]: It seems that it is not needed anymore

**Review by mrDIMAS — CHANGES_REQUESTED** [review]

File: src/rg3d/beginning/getting_started.md (hunk 1 above)
Comment (git2013vb) [reply src/rg3d/beginning/getting_started.md:10]: I deleted it before PR. You still see it?

Comment (mrDIMAS) [reply src/rg3d/beginning/getting_started.md:10]: Yeah, the file is still changed, I think you can revert changes from it.

**Comment by mrDIMAS** [comment]
Also please fix following errors from CI

```
error: character constant must be escaped: `'`
  --> /tmp/mdbook-5VIIPe/rg3d/ui/creating_button.md:165:2
   |
41 | '''
   |  ^ help: escape the character: `\'`

error[E0433]: failed to resolve: use of undeclared type `ControlFlow`
  --> /tmp/mdbook-5VIIPe/rg3d/ui/creating_button.md:153:29
   |
29 |             *control_flow = ControlFlow::Exit;
   |                             ^^^^^^^^^^^ use of undeclared type `ControlFlow`

error[E0412]: cannot find type `ControlFlow` in this scope
  --> /tmp/mdbook-5VIIPe/rg3d/ui/creating_button.md:151:76
   |
27 |     fn on_tick(&mut self, engine: &mut Engine, dt: f32, control_flow: &mut ControlFlow) {
   |                                                                            ^^^^^^^^^^^ not found in this scope
   |
help: consider importing one of these items
   |
3  | use rg3d::event_loop::ControlFlow;
   |
3  | use std::ops::ControlFlow;
   |

error[E0609]: no field `quit_button_handle` on type `&mut Game`
  --> /tmp/mdbook-5VIIPe/rg3d/ui/creating_button.md:159:46
   |
35 |             if message.destination() == self.quit_button_handle {
   |                                              ^^^^^^^^^^^^^^^^^^ unknown field
   |
   = note: available fields are: `exit`
```

**Comment by git2013vb** [comment]
I added the ControlFlow in use section. But same error... The row number match the one in file?

**Pull request merged**